packages/cbioportal-clinical-timeline/src/TrackHeader.tsx

```diff
@@ -44,6 +52,17 @@ const TrackHeader: React.FunctionComponent<ITrackHeaderProps> = function({
                 <span>
                     <TruncatedText text={getTrackLabel(track)} maxLength={20} />
                 </span>
+                {isLineChartTrack && (
```

Comment:
i would like to see if there's any way we can make this generic, so TrackHeader doesn't know about linechart type.   the problem is we need to be able to render a child of the track into this region, but  somehow from within the track component.  lets think on this.

Reply:
Im not sure I agree. so far the only thing we need to bring into the track headers is an axis. The generic route as far as I see is to make it more like doesTrackHaveAxis which we can do in the future if/when we add other types of tracks with axes. Or we could even embed that in the track specification regardless of track type